src/app/doubtfire-angularjs.module.ts

```diff
@@ -89,7 +89,6 @@ import 'build/src/app/projects/states/dashboard/directives/student-task-list/stu
 import 'build/src/app/projects/states/dashboard/directives/directives.js';
 import 'build/src/app/projects/states/dashboard/directives/task-dashboard/directives/task-assessment-card/task-assessment-card.js';
 import 'build/src/app/projects/states/dashboard/directives/task-dashboard/directives/task-outcomes-card/task-outcomes-card.js';
-import 'build/src/app/projects/states/dashboard/directives/task-dashboard/directives/task-submission-card/task-submission-card.js';
```

Comment:
This file should also be removed now this is migrated.

Reply:
The build folder is ignored and not checked in, but i have removed the files.